CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 #### Fixes
 
 * [#1365](https://github.com/ruby-grape/grape/pull/1365): Fix finding exception handler in error middleware - [@ktimothy](https://github.com/ktimothy).
+* #1367: `allow_blank: false` for `Time` attribute with valid value causes an error [@ipkes](https://github.com/ipkes).
```

Comment:
Please stick to the format of other CHANGELOG entries. Pay attention to what's a link, then name, the period at the end :)